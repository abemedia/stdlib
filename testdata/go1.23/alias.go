package test

import (
	xslices "golang.org/x/exp/slices" // want "Package \"golang.org/x/exp/slices\" can be replaced with \"slices\""

	"github.com/samber/lo"
	hi "github.com/samber/lo" // want "The github.com/samber/lo package import is no longer necessary"
)

func _(a []string) {
	lo.Keyify(a)
	hi.Drop(a, 2)      // want `lo.Drop can be replaced with builtin`
	hi.DropRight(a, 2) // want `lo.DropRight can be replaced with builtin`
	xslices.Clone(a)
}
