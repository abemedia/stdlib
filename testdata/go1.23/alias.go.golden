package test

import (
	"github.com/samber/lo"
	// want "The github.com/samber/lo package import is no longer necessary"
)

func _(a []string) {
	lo.Keyify(a)
	a[2:]        // want `lo.Drop can be replaced with builtin`
	a[:len(a)-2] // want `lo.DropRight can be replaced with builtin`
}
